source/_integrations/s3backup.markdown

```
@@ -0,0 +1,47 @@
+
+This integration follows standard integration removal. No extra steps are required.
+
+{% include integrations/remove_device_service.md %}
+
+## Troubleshooting
+
+{% details "Authentication failure" %}
+
+Make sure your credentials (access key/secret key) have read/write access to your bucket.
+
+{% enddetails %}
```

Markdownlint (0.37.0)

Files should end with a single newline character

(MD047, single-trailing-newline)